pkg/backend/display/rows.go

```diff
@@ -435,19 +435,14 @@ func (data *resourceRowData) getInfoColumn() string {
 }
 
 func getDiffInfo(step engine.StepEventMetadata, action apitype.UpdateKind) string {
-	diffOutputs := action == apitype.RefreshUpdate
```

Comment:
We are no longer treating a refresh as an "output diff".  It is now a diff of inferred inputs, preferably from the `detailedDiff` computed by the step.